providers/intercom/objectNames.go

```diff
@@ -18,9 +19,32 @@ var ObjectNameToResponseField = handy.NewDefaultMap(map[string]string{ //nolint:
 	"events":        "events",
 	"segments":      "segments",
 	"activity_logs": "activity_logs",
+	"tickets":       "tickets",
+	"conversations": "conversations",
 },
 	func(key string) string {
 		// Other objects are mapped to `data`.
 		return "data"
 	},
 )
+
+var objectNameToURLPath = handy.NewDefaultMap(map[string]string{ //nolint:gochecknoglobals
+	"activity_logs": "/admins/activity_logs",
+	"collections":   "/help_center/collections",
+	"help_centers":  "/help_center/help_centers",
+	"news_items":    "/news/news_items",
+	"newsfeeds":     "/news/newsfeeds",
+}, func(obj string) string {
+	return obj
+})
+
+var incrementalSearchObjectPagination = handy.NewDefaultMap(map[string]int{ //nolint:gochecknoglobals
```

Comment:
You can add `nolint:gomnd` on the line above the var and it should not lint the var (so that you don't have to repeat it)

Reply:
That worked!